inmanta.spec

```diff
@@ -176,7 +176,7 @@ touch %{buildroot}/etc/sysconfig/inmanta-server
 
 # Install web-console
 mkdir -p %{buildroot}/usr/share/inmanta/web-console
-tar -xf %{SOURCE3} --strip-components=2 --directory %{buildroot}/usr/share/inmanta/web-console
```

Comment:
So this make the time the time the rpm was built, not when the webconsole  was built?

Reply:
Correct